contracts/Strategy.sol

```diff
@@ -196,9 +195,7 @@ contract Strategy is BaseStrategy {
         if (_looseWant > _debtOutstanding) {
             uint256 _amountToDeposit = _looseWant.sub(_debtOutstanding);
 
-            if (_amountToDeposit > 0) {
```

Comment:
Should we not still be checking that this is !0?

Attempting to deposit 0 into stargate would revert?

Reply:
I cannot be 0